test/fuzz/test_runner.py

```diff
@@ -121,6 +136,22 @@ def main():
     )
 
 
+def merge_inputs(*, corpus, test_list, build_dir, merge_dir):
+    logging.info("Merge the inputs in the passed dir into the seed_dir. Passed dir {}".format(merge_dir))
```

Comment:
This looks like leftover debugging.

Reply:
Merging is currently a manual operation, so I like the extra verbosity. Happy to remove if you have strong feelings about this.

Reply:
If it's intentional, no problem; It just seemed written strangely as a log message, so I wanted to make sure it wasn't a leftover.